Gaijinizer patch/Map050.txt

```diff
@@ -421,9 +423,9 @@ dancing? \\N[1]-kun!#MTLed
 しーっかりダンスにも表れてて、すごく期待通りだよっ♪
 > CONTEXT: Map050/events/46/pages/0/255/Dialogue/enemy6
 > CONTEXT: Map188/events/46/pages/0/255/Dialogue/enemy6
-Its 'Despise temptation' 'Obstinacy to survive'
-desperate, It appears well in the dance and it's
-quite as expected♪#MTLed
+"Resisting temptation", a desperate "will to
```

Comment:
This will make it flow a bit nicer if we do a few line breaks:
>**_"Resisting temptation...."_**
>**_A desperate "will to survive..."_**
>**_It shows in your dance moves, just
as I expected. ♪_**